tonic/src/transport/server/mod.rs

```diff
@@ -595,7 +598,9 @@ impl<L> Server<L> {
                     let req_svc = svc
                         .call(&io)
                         .await
-                        .map_err(super::Error::from_source)?;
+                        .map_err(super::Error::from_source)?
```

Comment:
Style nit: how do you feel about using an explicit `match` here? I generally feel like `map().map_err()` doesn't give as much clarity.

Reply:
Personally, I feel the style using method chain like this to be a bit more intuitive.

Reply:
I feel the style using method chain like this to be a bit more intuitive.